src/controllers/authenticationController.ts

```diff
@@ -477,6 +499,11 @@ export default class AuthenticationController {
         // in io-spid-commons does not support 429 / 409 errors
         return ResponseErrorInternal(createProfileResponse.kind);
       }
+
+      userEmail =
+        newProfile.is_email_validated && newProfile.email
+          ? newProfile.email
+          : user.spid_email;
```

Comment:
This is always equal to the `spid_email` or undefined for CIE logins

Reply:
Right, done in [[#IOPID-385] just take spid_email for new users](https://github.com/pagopa/io-backend/pull/1040/commits/fb9fdd352426112daef92e210381128d2e2230c2)